p4-16/psa/PSA.mdk

```diff
@@ -1013,16 +984,43 @@ The `clone` flag specifies whether a packet should be cloned.  If
 true, then a cloned packet should be generated at the end of the
 pipeline.  The `clone_session_id` specifies one of several possible
 clone sessions that the control plane may configure in the PRE.  For
-each clone session, the control plane may configure the `egress_port`
-and `class_of_service` values that should be associated with packets
-cloned using that session.
+each clone session, the control plane may configure the following
+values that should be associated with packets cloned using that
+session.
+
+```
+PortId_t         egress_port;
+CloneSessionId_t class_of_service;
+bool             truncate;
+PacketLength_t   truncate_length_bytes;
+```
```

Comment:
@samar-abdi Take a look at these updated configuration parameters of a clone session to see if it looks reasonable.  egress_port and class_of_service were already there -- I've just made their PSA types explicit now.  truncate and truncate_length_bytes are new with this PR.

Reply:
Looks good. Thanks! I would suggest s/truncate_length_bytes/packet_length_bytes;

Reply:
If we go with @samar-abdi's renaming (which makes sense because is not how much we're truncating, but rather how much is left), I suggest we add a comment explaining the link to truncation. Alternatively, we can rename it to `truncated_packet_length` (and the comment that it is in bytes).

Reply:
I renamed it to packet_length_bytes.  Also added a note that a PSA implementation is allowed to support a restricted subset of possible values for this property of a clone session, e.g. a PSA implementation might only support multiples of 32 bytes.